dts/riscv/espressif/esp32c3.dtsi

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021 Espressif Systems (Shanghai) Co., Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <mem.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu0: cpu@0 {
+			device_type = "cpu";
+			compatible = "riscv";
+			reg = <0>;
+		};
+	};
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "simple-bus";
+		ranges;
+		
+		sram0: memory@3fc7c000 {
+			compatible = "espressif,esp32c3";
```

Comment:
This seems like a odd compatible.  Any reason this isn't `mmio-sram`?

Reply:
Probably this was inherited when I imported some lines from esp32 dts, I corrected it. 

Thanks!